2007-10-14  Ulrich Drepper  <drepper@redhat.com>

        * timezone/tst-timezone.c: Information in daylight and tzname does
        change for Asia/Tokyo timezone with more concrete information.
        Remove the test.

--- tzdata/src/tst-timezone.c.orig	2006-02-09 14:14:05.000000000 +0100
+++ tzdata/src/tst-timezone.c	2007-11-02 22:47:48.873604698 +0100
@@ -44,7 +44,6 @@
   { "America/Chicago", 1, 21600, {"CST", "CDT" }},
   { "America/Indiana/Indianapolis", 1, 18000, {"EST", "EDT" }},
   { "America/Los_Angeles", 1, 28800, {"PST", "PDT" }},
-  { "Asia/Tokyo", 1, -32400, {"JST", "JDT" }},
   { "Pacific/Auckland", 1, -43200, { "NZST", "NZDT" }},
   { NULL, 0, 0 }
 };
